_posts/namagic/2016-04-28-namagic-meetup.md

```diff
@@ -0,0 +1,16 @@
+---
+author: namagic
+layout: post
+title: "Omar's Revised Project Plan 4/21"
+---
+
+<h2>Intro</h2>
+I went to another TriPython event today. This one was all about regular expressions. This was a larger group, less beginners this round and mostly experienced developers. I think probably half of the people here were at another TriPython event a few weeks ago, so the continuity is nice. Learned about another cool workshop/conference. This one is at UNC and it’s called Data Matters from June 20-24. For introductions, we were asked for our name and favorite python library. Several people actually said regex!
+
+<h2>Regex Talk</h2>
+So this regex talk was done through Jupyter so the syntax was different I think. The speaker just kind of rolled through different ways to look for things using regex. He went through different cases like searching for all vowels, searching for slashes to break up dates, and searching for words that end or start with various letters. Regex is so powerful! I mean knew it was, but this was so specific. The speaker actually found a bug and he explained that for a bit. Honestly didn’t really understand this piece. Following this, much of the conversation shifted over to dna and how regex is used in regards to searching and splittig strings of dna.
```

Comment:
He was actually using a custom module called `regex` which is designed to replace the built-in `re` that we used.  Sometimes if the built-ins arent doing what developers need they write and publish their own modules.